File: C/program64.c
#include<stdio.h>


void SumFactorNonFactor(int iNo)
{
    if(iNo < 0)
    {
        iNo = - iNo;
    }

    int iCnt = 0, iSum1 = 0, iSum2 = 0 ;

    for(iCnt = 1; iCnt < iNo; iCnt++)
    {
        if((iNo % iCnt) != 0)
        {
            iSum1 = iSum1 + iCnt;  
        }
        else
        {
            iSum2 = iSum2 + iCnt;
        }

    }

    printf("Summation of Factor is %d\n",iSum2);
    printf("Summation of Non Factor is %d\n",iSum1);
}

int main()
{
    int iValue = 0;
   
    printf("Enter number :");
    scanf("%d",&iValue);

    SumFactorNonFactor(iValue);

    return 0;
}

File: Python/program64.py
class NonFactor:
    _No = 0
    
    def __init__(self, iNo):
        self._No = iNo
           
    def SumNonFactor(self):
        sum = 0
        for iCnt in range(1,self._No,1):
            if((self._No % iCnt) != 0): 
                sum = sum + iCnt

        return  sum
           
 
    
def main(): 
    
    iValue = int (input("Enter number : "))
        
    sobj = NonFactor(iValue)
    iRet = sobj.SumNonFactor()
    
    print("Summation of NON factor is ",iRet)

if __name__ == "__main__":
    main()